pkgs/development/python-modules/scalene/default.nix

```diff
@@ -10,9 +11,11 @@
   jinja2,
   numpy,
   psutil,
-  pynvml,
+  pynvml, # to be removed on next release
+  nvidia-ml-py,
   pytestCheckHook,
   pythonOlder,
+  python3,
```

Comment:
```suggestion
```

also I see this was formatted with `nixfmt-rfc-style` in 59b1aef59071cae6e87859dc65de973d2cc595c0, so let's stick to that

Reply:
Fixed in the update I just pushed.

[f08d1e0](https://github.com/NixOS/nixpkgs/commit/f08d1e080da168003a287f3c6bf7c9c7912d9d55)

Reply:
I updated my editor settings to use `nixfmt` (rfc style) from the old packages formatter.